emails/views.py

```diff
@@ -1634,6 +1729,17 @@ def _handle_complaint(message_json: AWS_SNSMessageJSON) -> HttpResponse:
         profile.auto_block_spam = True
         profile.save()
 
+    disable_mask_on_complaint_flag, _ = get_waffle_flag_model().objects.get_or_create(
+        name="disable_mask_on_complaint",
+        defaults={
+            "note": (
+                "MPP-3119: When a Relay user marks an email as spam, disable the mask."
+            )
+        },
+    )
+    if disable_mask_on_complaint_flag.is_active_for_user(user):
```

Comment:
I'd rather not use `is_active_for_user` until https://github.com/jazzband/django-waffle/pull/498 is released. Please use our override for a while longer:

```suggestion
    if flag_is_active_in_task("disable_mask_on_complaint_flag", user):
```

Reply:
Addressed in https://github.com/mozilla/fx-private-relay/pull/5045/commits/5d2e79df59aa660422e44018406d5793651e26e2